### ISSUE TITLE
RFC: integrate living styleguide generator for react components

### ISSUE THREAD

**Issue by dimaip** [neos/neos-ui#105]
https://github.com/pocotan001/react-styleguide-generator

I propose to keep the styleguide documentation files in the same folder as components, something like `docs.js`:

```
import React from 'react'
import Button from './Button'

export default class extends React.Component {
  static styleguide = {
    index: '1.1',
    category: 'Elements',
    title: 'Button',
    description: 'You can use **Markdown** within this `description` field.',
    code: `<Button size='small|large' onClick={Function}>Cool Button</Button>`,
    className: 'apply the css class'
  }

  onClick () {
    alert('Alo!')
  }

  render () {
    return (
      <Button size='large' onClick={this.onClick}>Cool Button</Button>
    )
  }
}
```

**Comment by dimaip** [comment]
![neos-ui](https://cloud.githubusercontent.com/assets/837032/14241740/c2109740-fa55-11e5-839c-7f48141bd8de.png)
Work by @GBKS.

**Comment by dimaip** [comment]
Closed by #153
